getting-started/glossary.md

```diff
@@ -1,51 +1,93 @@
-# GLOSSARY   
+# GLOSSARY
 
-## API
-Application Programming Interface. A software intermediary that allows two applications to talk to each other.
+This page provides a brief explanation of the key terms used in Cloud File Transfer  
 
+## API
+Application Programming Interface. Refers to a set of functions and procedures that allows two applications to interact with each other programmatically in order to access the features or data of an operating system, application or other service. 
 
 ## API Endpoint
-The place that APIs send requests to and where the resource resides.
+Refers to a URL of a server or service, that is, the location from which APIs can access the API resources that they need to carry out the function.
 
 ## API Gateway
-An API management tool that sits between a client and a collection of backend services.
-
-## API Gateway Id
-
+An API management tool that sits between a client and a collection of backend services. It accepts all application programming interface (API) calls, aggregates the various services required to fulfill them, and returns the appropriate result.
 
 ## API Key
-A unique identifier used by CFT to authenticate a calling program, this is used for rate limiting. It is assigned to an individual project.
+A unique identifier used by CFT to authenticate a calling program. It is assigned to an individual project and used for rate limiting.
 
 ## API Request
-A call to a server using an API.
+A call to a server using an API. It allows you to retrieve data from a data source, or to send data. APIs run on web servers, and expose endpoints to support the operations client applications use to provide their functionality. Each API request uses an HTTP method. The most common methods are GET , POST , PATCH , PUT , and DELETE.
+
+- GET method retrieves data from an API
+- POST method sends new data to an API
+- PATCH and PUT methods update existing data
+- DELETE removes existing data
 
 ## API Response
-Response sent by the server according to the input header of the API request.
+A response is defined by its HTTP status code and the data returned in the response body and/or headers.
+For example: 
+200 OK - (When the request completes successfully and returns the response body.)
+
+## AIAS
+Application Infrastructure Architecture Standard.
+
+## AWS
+Amazon Web Services. It is the world’s most comprehensive and broadly adopted cloud platform, offering over 200 fully featured services from data centers globally. 
 
 ## Client Id and Secret
+Refers to the API credentials, required to generate an authorization token to make API calls.
+
+## CFT
+Cloud File Transfer. A centralized, fully managed file transfer service for the whole of government (WOG). CFT is a part of the SG Tech Stack (SGTS) and is hosted on the Government Commercial Cloud (GCC).
```

Comment:
"A centralised, fully managed file transfer service for the whole-of-government (WOG). This service is part of the Singapore Tech Stack (SGTS) and is hosted on the Government Commercial Cloud (GCC).